docs/nzxt-hue2-guide.md

```diff
@@ -26,6 +26,12 @@ It provides two HUE 2 lighting channels and three independent fan channels with
 
 A microphone is still present onboard for noise level optimization through CAM and AI.
 
+## NZXT H1 V2 Smart Device
```

Comment:
```suggestion
## NZXT H1 V2
```

Reply:
Additionally, the docstring of the `smart_device` module should also get a small section for the H1 V2.